content/departments/product-engineering/process/planning-communication.md

```diff
@@ -0,0 +1,80 @@
+  - Any other helpful information (ex: risks, requests for help).
+- VP Product and VP Eng review and ask questions async and then anything that needs discussion is added to [VP team sync agenda](../team/index.md#vp-team-sync).
+
+## Roadmap
+
+The purpose of our roadmap is to communicate to our team and our customers the story of what we plan to deliver to acheive our our annual vision and quarterly OKRs.
+
+Expectations around roadmap items depend on how far in the future we expect them to happen. Our goal is to have:
+
+- High confidence and fidelity in our roadmap for the next 3 months on a rolling basis (reviewed/updated ~monthly)
+- Our best approximation for the biggest milestones we want to hit over the next 12 months on a rolling basis (reviewed/updated ~quarterly)
+
+The roadmap should always reflect the realistic expectations of the team
+
+- Don't invent roadmap items that the team has low confidence in just to fill space on the roadmap (at the same time it is important for teams to challenge themselves to think about what they want to aim to accomplish over a time horizon longer than one quarter).
+- Don't provide false precision on dates (if you can only place items into a quarter or month, that is fine)
+- Don't include stretch roadmap items (just adjust the timeline out so you don't feel the need to label something as "stretch")
+
+For FY23 Q1, the source of truth for our roadmap is [this GitHub project](https://github.com/orgs/sourcegraph/projects/214).
+
+- Each roadmap item should link to an issue that follows [this template](https://github.com/sourcegraph/sourcegraph/issues/new?template=roadmap-issue.md)
+- Teams should keep the delivery plan up-to-date and post updates to the issue anytime something important happens (ex: milestone achieved, plan changed)
+
+## Team newsletters
+
+Team newsletters communicate a high-level narrative/summary to the rest of the company about what each team in product/eng accomplished since the last newsletter and what is up next.
+
+The engineering manager of each team is responsible for sending an update to [engineering-team-status@sourcegraph.com](https://groups.google.com/a/sourcegraph.com/g/engineering-team-status) by the end of the week that contains the first day of each month. EMs may delegate to someone else on the team to send the update.
+
+Guidelines:
+
+- The email subject should contain the team name and the date (for example: "Search update 2020-10-14") so each update starts a distinct email thread (otherwise, they get grouped in the Google Groups UI).
+- There is no strict format to follow for the content, but here are some tips:
+  - Favor prose over a list of changes.
+  - Be brief and link to details.
+  - Consider sharing wins, challenges, risks, plans, lessons learned.
+  - Inline relevant screenshots/demos/gifs/charts when possible so it isn't just a wall of text.
+  - Be creative and have fun with it! Jokes and random fun facts are welcome.
+
+## Monthly PMM roadmap update
```

Comment:
This (and maybe some other content on this page?) duplicates some of the content in the product management handbook: https://handbook.sourcegraph.com/departments/product-engineering/product/process. In particular https://handbook.sourcegraph.com/departments/product-engineering/product/process#keeping-strategy-up-to-date.